### DISCUSSION TITLE
Is it possible to use capacitor with a Django application ?

### DISCUSSION THREAD

**Discussion by jassimabdullatheef** [ionic-team/capacitor#3488] [Q&A]
I have a Django application that loads a React app on a child page. Is it possible to use the ionic Capacitor with this setup?

**Comment by mlynch** [comment]
Yes, it is possible to use Capacitor APIs to turn that into a PWA. However, you won't do this to deploy to iOS or Android because apps must be locally contained and not load an external web app as a resource. Thus, you'd need to enable that React SPA to be able to run entirely "locally" without being rendered by the server. Does that make sense?
